kolibri/core/assets/src/content/utils.js

```diff
@@ -0,0 +1,27 @@
+function validateAssessmentMetaData(data) {
+  // Data is coming from a serializer for a one to many key, so at least will return an empty array.
```

Comment:
> so at least will return an empty array

I don't quite understand what this means. Is `data.assessmentmetadata` always an array with length 0 or 1?